src/block/progress-bar/edit.js

```diff
@@ -137,6 +139,9 @@ const Edit = props => {
 						</div>
 					</div>
 				</div>
+				<style>
+					{ `.editor-styles-wrapper .stk-${ props.attributes.uniqueId } .stk-progress-bar { --progress-value:${ derivedValue }% !important; }` }
+				</style>
```

Comment:
```suggestion
				{ /** Add our progress style here because we're adjusting the value using a hook */ }
				<style>
					{ `.editor-styles-wrapper .stk-${ props.attributes.uniqueId } .stk-progress-bar { --progress-value:${ derivedValue }% !important; }` }
				</style>
```